modules/serverless-installing-cli-linux-rpm.adoc

```diff
@@ -6,9 +6,11 @@
 [id="installing-cli-linux-rpm_{context}"]
 = Installing the Knative CLI for Linux using an RPM
 
-For {op-system-base-full}, you can install the `kn` CLI as an RPM if you have an active {product-title} subscription on your Red Hat account.
+For {op-system-base-full}, you can install the Knative (`kn`) CLI as an RPM. This allows the Knative CLI version to be automatically managed by the system. For example, using a command like `dnf upgrade` upgrades all packages, including `kn`, if a new version is available.
```

Comment:
The `dnf upgrade` command will require a YUM/DNF repo, which we are not providing. The RPM itself will not update itself.